src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let prev = initialValue;
+    let startIndex = 0;
+
+    if (arguments.length < 2) {
+      startIndex = 1;
+      prev = this[0];
+    }
```

Comment:
Try to do such assignments using ternary operators.
```suggestion
    const prev = (arguments.length < 2) ? this[0] : initialValue;
    const startIndex = (arguments.length < 2) ? 1 : 0;
```